src/envoy/utils/filter_state_utils.cc

```diff
@@ -44,6 +54,25 @@ absl::string_view getStringFilterState(
   return filter_state.getDataReadOnly<StringAccessor>(data_name).asString();
 }
 
+void setErrorFilterState(Envoy::StreamInfo::FilterState& filter_state,
+                         const google::rpc::Status& status) {
+  auto state = std::make_unique<RpcStatusWrapper>();
```

Comment:
if the wrapper has a copy constructor, these 2 lines can be combined into one

Reply:
Done